eo-runtime/src/main/eo/org/eolang/io/stdin.eo

```diff
@@ -0,0 +1,4 @@
++package org.eolang.io
++rt jvm org.eolang:eo-runtime:0.1.18
+
+[] > stdin /string
```

Comment:
@nlchar I think it's a wrong design. I suggest this one:

```
[] > stdin
  [] > nextLine /string
  [] > @ /string
```

I can read a single line with `stdin.nextLine` or I can take all of it with just `stdin`.

Make sense?

Reply:
@yegor256 done